claro.theme

```diff
@@ -118,16 +119,74 @@ function claro_preprocess_menu_local_action(array &$variables) {
 
   // We require Modernizr's touch test for button styling.
   $variables['#attached']['library'][] = 'core/modernizr';
-  $variables['#attached']['library'][] = 'core/modernizr';
 }
 
 /**
  * Implements hook_element_info_alter().
  */
-function claro_element_info_alter(&$type) {
+function claro_element_info_alter(&$types) {
   // We require Modernizr for button styling.
-  if (isset($type['button'])) {
-    $type['button']['#attached']['library'][] = 'core/modernizr';
+  if (isset($types['button'])) {
+    $types['button']['#attached']['library'][] = 'core/modernizr';
+  }
+
+  // @todo Refactor when https://www.drupal.org/node/3016343 is fixed.
+  if (isset($types['text_format'])) {
+    $types['text_format']['#pre_render'][] = '_claro_text_format_prerender';
+  }
+}
+
+/**
+ * Prerender callback for text_format elements.
+ */
+function _claro_text_format_prerender($element) {
+  // Add clearfix for filter wrapper.
+  $element['format']['#attributes']['class'][] = 'clearfix';
+  // Hide format select label visually.
+  $element['format']['format']['#title_display'] = 'invisible';
+
+  // Fix JS inconsistencies of the 'text_textarea_with_summary' widgets.
+  // @todo Remove when https://www.drupal.org/node/3016343 is fixed.
+  if (
+    !empty($element['summary']) &&
+    $element['summary']['#type'] === 'textarea'
+  ) {
+    $element['#attributes']['class'][] = 'js-text-format-wrapper';
+    $element['value']['#wrapper_attributes']['class'][] = 'js-form-type-textarea';
+  }
+  return $element;
+}
+
+/**
+ * Implements template_preprocess_filter_guidelines().
+ */
+function claro_preprocess_filter_guidelines(&$variables) {
+  // Fix filter guidelines selector issue of 'filter/drupal.filter'.
+  $variables['attributes']['class'][] = 'filter-guidelines-item';
+  $variables['attributes']['class'][] = 'filter-guidelines-' . $variables['format']->id();
```

Comment:
Shouldn't this be `filter-guidelines-item--$id`?

Reply:
See https://github.com/drupalux/claro/pull/68#discussion_r241565145, we have to keep them.